melodic/distribution.yaml

```diff
@@ -3338,6 +3338,22 @@ repositories:
       url: https://github.com/PickNikRobotics/rosparam_shortcuts.git
       version: melodic-devel
     status: developed
+  rplidar_ros:
+    doc:
+      type: git
+      url: https://github.com/Slamtec/rplidar_ros.git
+      version: master
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/Slamtec/rplidar_ros-release.git
+      version: 1.7.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Just checking: Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, please either remove this line or provide the necessary access as described in the wiki page.